### PR TITLE
Downstream 1.3.278 and remove no longer needed script dependencies

### PR DESCRIPTION
**Pull request by aqnuep**
@dgkoch, if you'd like to, please review the changes that resulted from downstreaming from the 1.3.278 tag of the Vulkan-Headers to update the build tooling. This PR will also remove the no longer necessary script dependencies.

### CODE REVIEW

File: registry/stripAPI.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+#
+# Copyright 2023 The Khronos Group Inc.
```
Comment (dgkoch) [inline registry/stripAPI.py:3]:
```suggestion
# Copyright 2023-2024 The Khronos Group Inc.
```
?

**Review by dgkoch — APPROVED** [review]
Overall seems fine. 

Are you planning to merge this now, or wait for the spec release?

File: registry/stripAPI.py (hunk 1 above)
Comment (aqnuep) [reply registry/stripAPI.py:3]: Funnily, this comes from the upstream repo :)

**Comment by aqnuep** [comment]
> Are you planning to merge this now, or wait for the spec release?

I am planning to merge this to main before the spec release, just to avoid having references to pending branches. These are all just changes coming from the upstream repo with no impact on the official Vulkan SC headers.

Do you have any concerns about that?

**Pull request merged**